queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth between '2016-01-01' and '2019-12-31';
+SELECT name FROM animals WHERE neutered = TRUE AND  escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name NOT IN ('Agumon', 'Pikachu');
```

Comment:
- [ ] As per requirement, `List date of birth of all animals named either "Agumon" or "Pikachu".` , We are supposed to list all date_of_birth of animals that are named Agumon or Pikachu we currently are listing all that is not named `Agumon` or `Pikachu`, Let's kindly remove `NOT`, I also like that you used `IN` instead of `name = 'Pikachu' OR name = 'Agumon';`

Reply:
Asked change implemented